src/Protocol/ActivityPub/Queue.php

```diff
@@ -132,30 +144,31 @@ public static function deleteById(int $id)
 	/**
 	 * Set the worker id for the queue entry
 	 *
-	 * @param array $activity
-	 * @param int   $wid
+	 * @param int $entryid
+	 * @param int $wid
 	 * @return void
 	 */
-	public static function setWorkerId(array $activity, int $wid)
+	public static function setWorkerId(int $entryid, int $wid)
```

Comment:
Please capitalize the `Id` in `$entryId`.
```suggestion
	public static function setWorkerId(int $entryId, int $wid)
```